apps/AEPSampleApp/App.tsx

```diff
@@ -2,6 +2,7 @@ import * as React from 'react';
 import { Button, View } from 'react-native';
 import { createDrawerNavigator } from '@react-navigation/drawer';
 import { NavigationContainer } from '@react-navigation/native';
+import Optimize from './extensions/Optimize';
```

Comment:
nit: can we rename it to `OptimizeView` to keep consistency with other extension pages?